### ISSUE TITLE
invalid path to js output

### ISSUE THREAD

**Issue by paulprok** [JeffreyWay/laravel-mix#1287]
here is mix file
```
let mix = require('laravel-mix');
mix
    .setPublicPath('public/lp')
    .js('src/js/bundle.js', 'assets/app.js')
    .sass('src/sass/bundle.scss', 'assets/app.css');
```
so afer compiling I get app.css here:

`\public\lp\assets\app.css`

but app.js is here

`\public\lp\public\lp\assets\app.js`

why so??

**Comment by alanondra** [comment]
I think there's something sort of weird going on with `setPublicPath`. Would something like this work for you in the meantime?

	let path = require('path');
	let mix = require('laravel-mix');

	let assetDir = 'lp/assets';

	// put images in assetDir
	mix.config.fileLoaderDirs.images = assetDir + path.sep + mix.config.fileLoaderDirs.images;

	// put fonts in assetDir
	mix.config.fileLoaderDirs.fonts = assetDir + path.sep + mix.config.fileLoaderDirs.fonts;

	mix.sass('src/sass/bundle.scss', 'public/%s/app.js'.replace('%s', assetDir));

	mix.js('src/js/bundle.js', 'public/%s/app.css'.replace('%s', assetDir));

**Comment by m2de** [comment]
Ran into this problem today as well. Just forked to commit a fix when I saw that Jeffery already fixed it in this commit [9a0cbbc](https://github.com/JeffreyWay/laravel-mix/commit/9a0cbbcdb38e3dceefb74d804ab4ac7318728738)
